Source/Fuse.Launcher.InAppBrowser/Trigger.uno

```diff
@@ -0,0 +1,38 @@
+using Uno;
+
+namespace Fuse.Triggers.Actions
+{
+
+	/** Launch the default map app
+
+		You'll find this trigger action in the Fuse.Launcher package, which have to be referenced from your uno project.
+		For example:
+
+			{
+				"Packages": [
+					"Fuse",
+					"FuseJS",
+					"Fuse.Launcher"
+				]
+			}
+
+		## Example
+
+			<StackPanel Margin="20">
```

Comment:
Please use `xml` code-block, or similar to https://github.com/fuse-open/fuselibs/commit/e885f5feee198a71a7f75a015b713e1c8d53af43#diff-e65a11c5d8ba79d36b1a9895f5c70b2dc8959e818f3e951a20da014e593e588dR145